lib/config/folderconfiguration.go

```diff
@@ -44,6 +43,7 @@ type FolderConfiguration struct {
 
 	DeprecatedReadOnly       bool    `xml:"ro,attr,omitempty" json:"-"`
 	DeprecatedMinDiskFreePct float64 `xml:"minDiskFreePct,omitempty" json:"-"`
+	DeprecatedPullerSleepS   int     `xml:"pullerSleepS" json:"pullerSleepS"`
```

Comment:
I think remove this, as it's not used anywhere.

Reply:
Ok. Just for my information: What is the purpose of keeping any of these deprecated variables?

Reply:
Usually we keep them to be able to migrate old configuration. If there is nothing to migrate we don't need to keep it.